app/src/main/java/io/intrepid/contest/models/Contest.java

```diff
@@ -82,6 +86,14 @@ public void setEntries(List<Entry> entries) {
         this.entries = entries;
     }
 
+    public void setEndedAt(String endedAt) {
```

Comment:
So it looks like we're setting these two variables but never accessing them. Do we actually need them?